### PR TITLE
Import log file functionality from p9p acme

### PR DESCRIPTION
**Pull request by floren**
With permission from Russ Cox.

### CODE REVIEW

File: sys/src/cmd/acme/logf.c [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+ *	- called from zeroxx
+ *
+ * op == "get" for Get executed on window
+ *	- called from get
+ *
+ * op == "put" for Put executed on window
+ *	- called from put
+ *
+ * op == "del" for deleted window
+ *	- called from winclose
```
Comment (fhs) [inline sys/src/cmd/acme/logf.c:156]: `op == "focus"` is missing. It's in a separate commit: https://github.com/9fans/plan9port/commit/fdf6ef333705c844bcf3ccf2f93b2773f1a6aa41

**Review by fhs — CHANGES_REQUESTED** [review]
The man page changes are missing.

File: sys/src/cmd/acme/logf.c (hunk 1 above)
Comment (floren) [reply sys/src/cmd/acme/logf.c:156]: that was imported in my second commit

**Comment by floren** [comment]
PTAL

**Review by fhs — APPROVED** [review]
:+1:

**Review by rminnich — APPROVED** [review]

**Review by rminnich — APPROVED** [review]

**Comment by floren** [comment]
i've made a mess of this with dumb rebasing, starting over.

**Pull request closed without merging**